Microsoft.Toolkit.Uwp.UI.Behaviors/VoiceCommands/WindowsMediaSpeechRecognizer.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using Windows.Globalization;
+using Windows.Media.SpeechRecognition;
+using Windows.UI.Core;
+
+namespace Microsoft.Toolkit.Uwp.UI.Behaviors
+{
+    public class WindowsMediaSpeechRecognizer : ISpeechRecognizer
+    {
+        private readonly SpeechRecognizer _speechRecognizer;
+
+        private WindowsMediaSpeechRecognizer(SpeechRecognizer speechRecognizer)
+        {
+            _speechRecognizer = speechRecognizer;
+        }
+
+        public static async Task<WindowsMediaSpeechRecognizer> CreateAsync(Windows.UI.Xaml.Window window)
+        {
+            SpeechRecognizer sr = null;
+            foreach (var item in Windows.System.UserProfile.GlobalizationPreferences.Languages)
+            {
+                var language = new Language(item);
```

Comment:
Should the language be set-able in the voice command as well (for multi-language use cases) and by default use this?

Reply:
Don't think Language should be a property on the VoiceCommandTrigger. Would love to know what others think though. It's all new for me too.